hack/check-registered-rest.sh

```diff
@@ -0,0 +1,16 @@
+set -x 
+
+#!/bin/bash
+fail() {
+	echo "$1"
+	exit 1
+}
+
+registerFuncs=$(cat api/api.pb.go | grep 'func Register.*' | wc -l)
+registered=$(cat api/server/sdk/rest_gateway.go | grep 'Register.*Handler,' | wc -l)
```

Comment:
Love this!

Reply:
Uh.. I would have used `egrep` or `grep -E` -- since I'm not sure "regular grep" will normally respect `.*`-regexp

Also, could reduce 3 commands into 1 ;) --> `grep -E 'Register.*Handler,' api/server/sdk/rest_gateway.go`